packages/xdg_base.rb

```diff
@@ -36,12 +37,12 @@ def self.postinstall
       system "echo '# See https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html' >> ~/.bashrc"
       system "echo 'export XDG_DATA_HOME=#{HOME}/.local/share' >> ~/.bashrc"
       system "echo 'export XDG_CONFIG_HOME=#{HOME}/.config' >> ~/.bashrc"
-      system "echo 'export XDG_DATA_DIRS=#{CREW_PREFIX}/share' >> ~/.bashrc"
+      system "echo 'export XDG_DATA_DIRS=#{CREW_PREFIX}/share:#{HOME}/.local/share/flatpak/exports/share:#{CREW_PREFIX}/var/lib/flatpak/exports/share' >> ~/.bashrc"
```

Comment:
Should these paths be added in the postinstall section of flatpak instead of this package?

Reply:
hmmm... That's a good idea.